run_test.go

```diff
@@ -77,17 +77,13 @@ func TestRunLifeCycle(t *testing.T) {
 
 	var leaderAddr raft.ServerAddress = "198.18.0.1:8300"
 
-	startTime := time.Date(2020, 11, 2, 12, 0, 0, 0, time.UTC)
-	nextStateTime := time.Date(2020, 11, 2, 12, 0, 0, 10000, time.UTC)
-	blockedStartTime := time.Date(2021, 1, 25, 16, 0, 0, 10000, time.UTC)
+	firstStateTime := time.Date(2020, 11, 2, 12, 0, 0, 0, time.UTC)
 
-	mtime.On("Now").Return(startTime).Once()
-	mtime.On("Now").Return(nextStateTime).Once()
-	mtime.On("Now").Return(blockedStartTime).Once()
```

Comment:
We used to get the current time in the first `Start` invocation, then when computing the next state and finally when trying to restart things after being stopped.

With the removal of tracking the start time and changing how we track the first state time, we now just get the current time when computing the next state.